$NetBSD: patch-libgcobol_configure.tgt,v 1.2 2026/07/19 11:17:45 adam Exp $

Support Darwin/aarch64.
https://github.com/Homebrew/homebrew-core/blob/main/Patches/gcc/gcc-15.3.0.diff

Enable libgcobol on NetBSD amd64, arm64, and riscv64.

--- libgcobol/configure.tgt.orig	2026-06-12 06:09:10.070579374 +0000
+++ libgcobol/configure.tgt
@@ -26,7 +26,7 @@
 LIBGCOBOL_SUPPORTED=no

 case "${target}" in
-    aarch64*-*-linux*)
+    aarch64*-*-linux* | aarch64-*-darwin* | aarch64*-*-netbsd*)
 	LIBGCOBOL_SUPPORTED=yes
 	;;
     powerpc64le-*-linux*)
@@ -34,12 +34,12 @@ case "${target}" in
 		LIBGCOBOL_SUPPORTED=yes
 	fi
 	;;
-    riscv64-*-linux*)
+    riscv64-*-linux* | riscv64-*-netbsd*)
 	if test x$ac_cv_sizeof_void_p = x8; then
 		LIBGCOBOL_SUPPORTED=yes
 	fi
 	;;
-    x86_64-*-linux* | i?86-*-linux* | x86_64-*-darwin*)
+    x86_64-*-linux* | i?86-*-linux* | x86_64-*-darwin* | x86_64-*-netbsd*)
 	if test x$ac_cv_sizeof_void_p = x8; then
 		LIBGCOBOL_SUPPORTED=yes
 	fi

